_posts/2023-12-11-notes-from-community-call-7-december-2023.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Notes from the Standard for Public Code community call - 7 December 2023"
+date: 2023-12-11
+author: Jan Ainali
+type: blogpost
+excerpt: Clarifying about different types of translations
+categories:
+  - Community call
+---
+
+# 7 December 2023 Standard for Public Code community call
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* [Elena Findley-de Regt](https://publiccode.net/who-we-are/team/elena-findley-de-regt.html)
+* Matti Schneider
+
+## Updates from the Foundation
+
+* We'll be running [a devroom at FOSDEM](https://blog.publiccode.net/news/2023/11/13/fosdem-2024-public-code-and-digital-public-goods-devroom-call-for-proposal.html)
+* We completed an assessment of [Wekan](https://wekan.github.io/standard-for-public-code/)
+
+## Notes
+
+### Background
+
+In the [Standard for Public Code](https://standard.publiccode.net/), we have [a requirement about translations](https://standard.publiccode.net/criteria/use-plain-english.html#requirements) stating "Any translation MUST be up to date with the English version and vice versa.".
+However, many open source projects have community contributed translations, and it is obviously not the intention of the standard to discourage this.
+
+Therefore, we had created a [proposal for change](https://github.com/publiccodenet/standard/pull/999) to address this.
+
+During the call we discussed this proposal and what it implies.
+One important observation was that it may not matter if it is the maintainers doing the translations or the community.
+Rather, it would be better to use terms like authoritative translations and courtesy translations.
```

Comment:
```suggestion
Rather, it would be better to use terms like authoritative translations - for translations the maintainers have committed to - and courtesy translations, for community contributed or automatic machine translations.
```